common_utils.py

```diff
@@ -221,21 +221,24 @@ def dragMoveEvent(self, event):
     def dropEvent(self, event):
         data = event.mimeData()
         mime = "text/uri-list"
-        url = str(data.data(mime))
-        path = urlparse(url).path.strip()
-        scheme = urlparse(url).scheme
+        url = urlparse(str(data.data(mime), encoding='utf-8').rstrip())
+        path = url.path.strip()
```

Comment:
Is the `.strip()` needed here is we'll employ it on the `url` variable above?

Reply:
I see this change was removed. I guess that's fair considering that now file dialog is being used and no extra line endings are added.